multi-watch/settingmanager.cpp

```diff
@@ -0,0 +1,46 @@
+#include "settingmanager.h"
+
+SettingManager::SettingManager()
+{
+
+}
+
+void SettingManager::saveTabOrder(QTabWidget *tabWidget)
+{
+  QSettings settings("multiWatchSetting", "multiWatch");
+  int nTab = tabWidget->count();
+  QString tabOrderTxt = "";
+  int i;
+  tabOrderTxt.append(tabWidget->tabText(0));
```

Comment:
Write code comments when you apply some logic that is not understandable at a first glance.
Comment about why you are keeping out the first one, and what you are dong in the loop